diff -Nur old/authlib/configure.in new/authlib/configure.in
--- old/authlib/configure.in	2003-10-25 03:02:55.000000000 +0000
+++ new/authlib/configure.in	2004-02-23 13:33:49.000000000 +0000
@@ -384,14 +384,6 @@
 	saveLIBS="$LIBS"
 	LIBS="$NETLIBS $LIBS"
 	LDAPLIBS=""
-        AC_MSG_CHECKING(whether -lresolv is needed for res_query)
-        AC_TRY_LINK_FUNC(res_query,
-        	AC_MSG_RESULT(no),
-        	LIBS="-lresolv $LIBS"
-        	AC_TRY_LINK_FUNC(res_query,
-        		LDAPLIBS="-lresolv $LDAPLIBS" HAVE_LDAP=1
-        		AC_MSG_RESULT(yes),
-        		AC_MSG_ERROR(Cannot find function res_query)))
 	AC_CHECK_LIB(lber, ber_scanf,
 		[ LDAPLIBS="-llber $LDAPLIBS" LIBS="-llber $LIBS" ])
 	AC_CHECK_LIB(ldap, ldap_open,
