--- sqwebmail-3.5.0/authlib/authmysql.h.orig	Wed Aug  8 05:47:38 2001
+++ sqwebmail-3.5.0/authlib/authmysql.h	Thu Feb 20 01:02:21 2003
@@ -5,8 +5,8 @@
 #include	<stdlib.h>
 #include	<string.h>
 #include	<sys/types.h>
-#include	<mysql.h>
-#include	<errmsg.h>
+#include	<mysql/mysql.h>
+#include	<mysql/errmsg.h>
 
 struct authmysqluserinfo {
 	char *username;
--- sqwebmail-3.5.0/authlib/authmysqllib.c.orig	Sun Jan 26 23:01:56 2003
+++ sqwebmail-3.5.0/authlib/authmysqllib.c	Thu Feb 20 01:03:41 2003
@@ -10,7 +10,7 @@
 #include	<ctype.h>
 #include	<sys/types.h>
 #include	<sys/stat.h>
-#include	<mysql.h>
+#include	<mysql/mysql.h>
 #include	<time.h>
 
 #include	"authmysql.h"
