diff -Nur old/configure.in new/configure.in
--- old/configure.in	2004-05-16 02:37:05.000000000 +0000
+++ new/configure.in	2004-06-01 22:32:22.000000000 +0000
@@ -280,6 +280,7 @@
 test "x$exec_prefix" = xNONE && exec_prefix='${prefix}'
 eval "prefix=$prefix"
 eval "libexecdir=$libexecdir"
+eval "DIR=$libexecdir"
 eval "localstatedir=$localstatedir"
 
 AC_ARG_WITH(makedatprog, [], ,
@@ -325,4 +326,4 @@
 	makedat rfc822 rfc2045 maildir soxwrap \
 	authlib http11 cgi ldapaddressbook \
 	gpglib pcp sqwebmail)
-AC_OUTPUT(Makefile dbobj.h dbobj.config sqwebmail.spec sqwebmaild.rc)
+AC_OUTPUT(Makefile dbobj.h dbobj.config sqwebmail.spec sqwebmaild.rc sqwebmail.init)
