diff -Nur old/pcp/pcpd.c new/pcp/pcpd.c
--- old/pcp/pcpd.c	2003-04-27 17:48:03.000000000 +0000
+++ new/pcp/pcpd.c	2004-04-15 11:44:16.000000000 +0000
@@ -2199,10 +2199,10 @@
 		exit(1);
 	}
 
-	if (chdir(a->maildir && *a->maildir ? a->maildir:"Maildir") < 0)
+	if (chdir(a->maildir && *a->maildir ? a->maildir:"Mail/Maildir") < 0)
 	{
 		free(token);
-		syslog(LOG_NOTICE, "pcpd: chdir(Maildir) failed.");
+		syslog(LOG_NOTICE, "pcpd: chdir(Mail/Maildir) failed.");
 		maildir_cache_cancel();
 		exit(1);
 	}
@@ -2211,7 +2211,7 @@
 	if (chdir("calendar") < 0)
 	{
 		free(token);
-		syslog(LOG_NOTICE, "pcpd: chdir(Maildir) failed.");
+		syslog(LOG_NOTICE, "pcpd: chdir(Mail/Maildir) failed.");
 		maildir_cache_cancel();
 		exit(1);
 	}
diff -Nur old/sqwebmail/auth.c new/sqwebmail/auth.c
--- old/sqwebmail/auth.c	2004-02-22 22:39:31.000000000 +0000
+++ new/sqwebmail/auth.c	2004-04-15 11:43:40.000000000 +0000
@@ -84,7 +84,7 @@
 static int login_maildir(const char *maildir)
 {
 	if (!maildir || !*maildir)
-		maildir="Maildir";
+		maildir="Mail/Maildir";
 	if (chdir(maildir))	return (-1);
 	maildir_loginexec();
 	return (0);
@@ -112,7 +112,7 @@
 char	*buf;
 struct stat stat_buf;
 
-	if (!p || !*p)	p="Maildir";
+	if (!p || !*p)	p="Mail/Maildir";
 
 	buf=malloc(strlen(h)+strlen(p)+sizeof(PASSFILE)+100);
 
diff -Nur old/sqwebmail/sqwebmail.h new/sqwebmail/sqwebmail.h
--- old/sqwebmail/sqwebmail.h	2004-02-29 20:30:56.000000000 +0000
+++ new/sqwebmail/sqwebmail.h	2004-04-15 11:43:45.000000000 +0000
@@ -25,7 +25,7 @@
 
 /* Location of the user's Maildir */
 
-#define USER_DIR	"Maildir"
+#define USER_DIR	"Mail/Maildir"
 
 /* For PAM-based authentication */
 
