diff -Nur old/pcp/pcpd.c new/pcp/pcpd.c
--- old/pcp/pcpd.c	2004-05-02 15:57:31.000000000 +0000
+++ new/pcp/pcpd.c	2004-06-01 22:27:02.000000000 +0000
@@ -2187,10 +2187,10 @@
 		exit(1);
 	}
 
-	if (chdir(a->maildir && *a->maildir ? a->maildir:"Maildir") < 0)
+	if (chdir(a->maildir && *a->maildir ? a->maildir:"Mail/Maildir") < 0)
 	{
 		free(token);
-		fprintf(stderr, "NOTICE: chdir(Maildir) failed: %s\n",
+		fprintf(stderr, "NOTICE: chdir(Mail/Maildir) failed: %s\n",
 			strerror(errno));
 		maildir_cache_cancel();
 		exit(1);
diff -Nur old/sqwebmail/auth.c new/sqwebmail/auth.c
--- old/sqwebmail/auth.c	2004-05-09 03:32:01.000000000 +0000
+++ new/sqwebmail/auth.c	2004-06-01 22:25:57.000000000 +0000
@@ -82,7 +82,7 @@
 	if (!maildir || !*maildir)
 		maildir=getenv("MAILDIRPATH");
 	if (!maildir || !*maildir)
-		maildir="Maildir";
+		maildir="Mail/Maildir";
 	if (chdir(maildir))	return (-1);
 	maildir_loginexec();
 	return (0);
@@ -111,7 +111,7 @@
 struct stat stat_buf;
 
 	if (!p || !*p)	p=getenv("MAILDIRPATH");
-	if (!p || !*p)	p="Maildir";
+	if (!p || !*p)	p="Mail/Maildir";
 
 	buf=malloc(strlen(h)+strlen(p)+sizeof(PASSFILE)+100);
 
diff -Nur old/sqwebmail/sqwebmaild.dist.in new/sqwebmail/sqwebmaild.dist.in
--- old/sqwebmail/sqwebmaild.dist.in	2004-05-09 03:32:01.000000000 +0000
+++ new/sqwebmail/sqwebmaild.dist.in	2004-06-01 22:22:10.000000000 +0000
@@ -32,7 +32,7 @@
 #
 # MAILDIRPATH - directory name of the maildir directory.
 
-MAILDIRPATH=Maildir
+MAILDIRPATH=Mail/Maildir
 
 ##NAME: AUTHMODULES:0
 #
diff -Nur old/sqwebmail/sqwebmail.h new/sqwebmail/sqwebmail.h
--- old/sqwebmail/sqwebmail.h	2004-02-29 20:30:56.000000000 +0000
+++ new/sqwebmail/sqwebmail.h	2004-04-15 11:43:45.000000000 +0000
@@ -25,7 +25,7 @@
 
 /* Location of the user's Maildir */
 
-#define USER_DIR	"Maildir"
+#define USER_DIR	"Mail/Maildir"
 
 /* For PAM-based authentication */
 
