--- sqwebmail-5.9.2/configure.ac.orig	2018-03-24 21:48:34.296563716 +0100
+++ sqwebmail-5.9.2/configure.ac	2018-03-24 21:49:01.836563402 +0100
@@ -304,4 +304,4 @@
 	libs/rfc822 libs/rfc2045 libs/maildir libs/soxwrap
 	libs/http11 libs/cgi libs/ldapaddressbook
 	libs/gpglib libs/pcp libs/sqwebmail)
-AC_OUTPUT(Makefile dbobj.h dbobj.config sqwebmail.spec sqwebmail.lpspec sqwebmaild.rc)
+AC_OUTPUT(Makefile dbobj.h dbobj.config sqwebmail.spec sqwebmail.lpspec sqwebmaild.rc sqwebmail.init)
