--- old/libs/sqwebmail/sqwebmail.c	2003-11-18 02:42:44.000000000 +0000
+++ new/libs/sqwebmail/sqwebmail.c	2004-02-20 16:27:39.000000000 +0000
@@ -2759,7 +2759,7 @@
 				((rm=getenv("CONTENT_TYPE")) != 0 &&
 				strncasecmp(rm,"multipart/form-data;", 20)
 					== 0)))
-			emsg("multipart/formdata posts not allowed","");
+//			emsg("multipart/formdata posts not allowed","");
 
 		/* Some additional safety checks */
 
