--- sqwebmail-3.5.0/sqwebmail/sqwebmail.c.orig	Sun Jan  5 05:26:06 2003
+++ sqwebmail-3.5.0/sqwebmail/sqwebmail.c	Thu Feb 20 20:40:43 2003
@@ -2375,7 +2375,7 @@
 				((rm=getenv("CONTENT_TYPE")) != 0 &&
 				strncasecmp(rm,"multipart/form-data;", 20)
 					== 0)))
-			enomem();	/* multipart/formdata posts not allowed
+			/*enomem();*/	/* multipart/formdata posts not allowed
 					*/
 
 		/* Some additional safety checks */
